include_core/omrport.h

```diff
@@ -566,6 +566,11 @@ typedef struct J9MemoryInfo {
 	uint64_t cached;			/* The physical RAM used as cache memory (in bytes). */
 	uint64_t buffered;			/* The physical RAM used for file buffers (in bytes). */
 	int64_t timestamp;			/* Sampling timestamp (in microseconds). */
+#if defined(LINUX) && !defined(OMRZTPF)
+	uint64_t hostAvailPhysical;	/* Available physical memory on the host (in bytes) when process is in a cgroup */
```

Comment:
Can you add something to the effect of `When not in a cgroup, this will be identical to memInfo->availPhysical` for each of the new fields?